src/primitives/mod.rs

```diff
@@ -26,6 +26,11 @@ pub enum Bech32 {}
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub enum Bech32m {}
 
+/// The codex32 checksum algorithm, defined in [BIP-93].
+/// [BIP-350]: <https://github.com/bitcoin/bips/blob/master/bip-0093.mediawiki>
```

Comment:
Should this be `[BIP-93]` here?

Reply:
Thanks, odd that `-D rustdoc::broken-intra-doc-links` didn't catch that. I've moved the codex32 stuff to a rustdoc comment and remove the link altogether.